ManatiCR/manatime
# 18: Projects and collaborators summary report.

**Steps to test**
- **To update aquifer**
  - Removed `node_modules` and `vendor` folders.
  - Run `composer install`.
  - Run `cd .vendor/drush/drush ; composer install`.
  - Run `npm install`.
  - Run `node_modules/.bin/aquifer build`.
  - Run `./scripts/manatime_local_install.sh`.
- **Functional test**
  - **Add content as an admin:**
  - Add some users with the role collaborator.
  - Add minimum two project nodes, each project must have tasks related to it and a client (It doesn't  matter if it's the same client for the projects).
  - Add some timesheet nodes (one for each of the collaborators) and add some time entries since there.
  - Change the time entries's author for the collaborator wich is related via timesheet.
  - **As an administrator user (not admin user):**
  - Go to `reports/summary/projects`, `reports/summary/collaborators` and see if the filters works.
  - The last thing, check if the `csv` button download a csv file with the data the report has and check if the `PDF` button goes to the respective URL and if the download is possible there. 

**DoD**
- Non-editable table with information per collaborator or per project.
- It should have filters: Per Project, Per collaborator, Per dates.
- It should be exportable as csv (stretch goal: pdf)
- Maybe 2 different views? (per project/per collaborator
- Per Project view: filters: date, collaborator. Fields: project, total time
- Per Collaborator view: filters: date, project. Fields: Collaborator, total time

[Issue link](https://github.com/ManatiCR/manatime/issues/18)
